### ISSUE TITLE
Some part of the admin page content is hidden due to styling issue

### ISSUE THREAD

**Issue by rahulkundu1209** [I-TECH-UW/OpenELIS-Global-2#1365]
1.  Admin

2. Open admin page and then open any page from the menu(like, Test Management -> Reflexive Test Management or Analyzer Test Name).

3. https://localhost/MasterListsPage#reflex

4. Screenshot
![image](https://github.com/user-attachments/assets/11c46cb7-d2d9-4c0a-8816-785bc66f20b2)

5. Indicate the severity of the issue using the following ratings:
   1. This issue is something we would like to see improved in the future but
      does not significantly affect our work.

*I want to fix this small styling issue as my first contribution.*

**Comment by harshitg927** [comment]
I am unable to reproduce this issue on my localmachine, is this is issue still there in the latest commit?

**Comment by rahulkundu1209** [comment]
Yes, it is still there. The issue related to responsiveness is clearly visible on screen width <1300px.
